css/css-grid_image-gallery/css/styles.css

```
@@ -41,6 +40,9 @@ body {
 .teaser-grid {
   min-width: 40rem;
   min-height: 100vh;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
```

not saving much letters I guess :)

```suggestion
  grid-template-columns: 1fr 1fr;
```